### PR TITLE
RSI 0102 and 0112 Increase test coverage of the downstream data

### PR DESCRIPTION
**Pull request by yoganandkunche**
### What is the context of this PR?
Dumps the answers that are going to be submitted and compares with the answers that were entered for both RSI monthly and quarterly (0102 & 0112).

### How to review 
Run the integration test.

### CODE REVIEW

File: tests/integration/rsi/test_rsi_change_answers.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+        # We proceed to the questionnaire
+        self.post(action='start_questionnaire')
+
+        # We are in the Questionnaire
+        self.assertInPage('>Monthly Business Survey - Retail Sales Index</')
+        self.assertInPage('What are the dates of the period that you will be reporting for?')
+        self.assertInPage('>Save and continue<')
+
+        if form_type_id is '0102':
```
Comment (collisdigital) [inline tests/integration/rsi/test_rsi_change_answers.py]: split this into two tests (duplication for the first part is OK)

File: tests/integration/rsi/test_rsi_change_answers.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+            'period-to-month': '4',
+            'period-to-year': '2016',
+        }
+
+        # We submit the form
+        self.post(form_data)
+        self.post(post_data={'total-retail-turnover-answer': '100'})
+        self.post(post_data={'internet-sales-answer': '50'})
+        self.post(post_data={'changes-in-retail-turnover-answer': 'Down streams data test'})
+        self.post(post_data={'male-employees-over-30-hours': '1'})
```
Comment (collisdigital) [inline tests/integration/rsi/test_rsi_change_answers.py]: I think all of these are on the same screen, i.e. should be posted together as a single .post like the period-to/from

File: tests/integration/rsi/test_rsi_change_answers.py [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+        actual = self.dumpSubmission()
+
+        expected = {
+            "submission": {
+                "data": {
+                    "11": "01/04/2016",
+                    "12": "30/04/2016",
+                    "20": "100",
+                    "21": "50",
+                    "50": "5",
```
Comment (collisdigital) [inline tests/integration/rsi/test_rsi_change_answers.py]: Where are the male/female split numbers (male-employees-under-30-hours) etc?

Comment (yoganandkunche) [reply tests/integration/rsi/test_rsi_change_answers.py]: I don't think we submit those fields to downstream as they do not have q_codes. I will double check with jenny

Comment (yoganandkunche) [reply tests/integration/rsi/test_rsi_change_answers.py]: sorry, my fault as I had multiple posts for a single page it has not entered all the values in the employees.  Made the changes and is not working as expected.

File: tests/integration/rsi/test_rsi_change_answers.py (hunk 2 above)
Comment (yoganandkunche) [reply tests/integration/rsi/test_rsi_change_answers.py]: Made the changes now pushing it up.

File: tests/integration/rsi/test_rsi_change_answers.py [hunk 4]
```diff
@@ -0,0 +1,168 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class RsiHappyPath(IntegrationTestCase):
```
Comment (collisdigital) [inline tests/integration/rsi/test_rsi_change_answers.py]: Our class names for Tests usually start with Test and should probably be more descriptive e.g. `TestRsiSubmissionData`

**Comment by collisdigital** [comment]
this is nearly there, but the filenames of the tests aren't right, they should match the test class names really.

**Comment by yoganandkunche** [comment]
Getting used to the coding standards and namings. i am getting there.!

**Review by collisdigital — APPROVED** [review]
Needs squashing and re-basing

**Review by samiwel — APPROVED** [review]
LGTM

**Pull request merged**